### DISCUSSION TITLE
Weekly report 12.10. - 19.10.23

### DISCUSSION THREAD

**Discussion by TadaXD20** [SE-TINF22B6/Plapy#47] [Weekly reports]
# Team Introduction
| Resource | GitHub name | Worker | Activities |
| ---------- | ------------- | ------- | ---------|
| Nick Möhrmann | Shiny1798 | Product Owner | coordinating tasks, feature implementation |
| Nils Bohland | Umbranecis | Database architect | designing database, connection implementation |
| Margot Geisbauer | QuinnyPikaPika | UI Designer | creating mockup for future web version, feature implementation |
| Tanina Blucha | TadaXD20 | Database administrator | implementing and managing schema and tables of database, feature implementation |

# Vision
We want to create a music Discord bot, to play playlists from YouTube and Soundcloud. We also want to contribute the possibility to create and save Discord playlists and adding currently playing songs to it.

# Techniques/platforms

- Discord
- Express
- NodeJS
- PM2
- PostgreSQL
- Soundcloud API
- TypeScript
- YouTube API

# Scrum setup

- Code management & documentation: GitHub
- Weekly: every Tuesday at 12:00 am, 15 minutes long
- Review + Retro: every Friday at 12:00 am, 1 hour long

# Sprint 1 backlog

![Sprint1_Backlog](https://github.com/SE-TINF22B6/Plapy/assets/75337582/f0799e97-1ddf-4a1f-9a95-11f7864f5ea8)

# Kanban

![Sprint1_Kanban_1](https://github.com/SE-TINF22B6/Plapy/assets/75337582/081a2d96-4999-4d78-8847-81f056697df8)
![Sprint1_Kanban_2](https://github.com/SE-TINF22B6/Plapy/assets/75337582/6912c01e-d2f5-40a1-81d9-798abf46f9a9)
![Sprint1_Kanban_3](https://github.com/SE-TINF22B6/Plapy/assets/75337582/88b200d1-4add-4eaf-91da-9251a09b8a80)

**Comment by Stev3eM** [comment]
Hey Plapys,

I like the structure of your secon blog post. The team, vision and technologies is 100% clear.  But you have to keep an eye on your data storage or scripting preview. As you can see in part _Sprint 1 backlog_ and _Kanban_ the hyperlink isn't working anymore.

Keep on going!
